csharp/src/Apache.Arrow.Adbc.FlightSql/Apache.Arrow.Adbc.FlightSql.csproj

```
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+    <AssemblyName>$(MSBuildProjectName)</AssemblyName>
+    <RootNamespace>Apache.Arrow.Adbc.FlightSql</RootNamespace>
+    <Version>0.1.0</Version>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Apache.Arrow.Flight" Version="12.0.0" />
+    <PackageReference Include="Grpc.Net.Client.Web" Version="2.53.0" />

```

```suggestion
    <PackageReference Include="Grpc.Net.Client" Version="2.53.0" />
```

I assume you don't really need the `.Web` package. If you do, why?

---

good catch. this was left over from getting things to work with .NET Standard. removed